webi-docker.sh

```diff
@@ -0,0 +1,81 @@
+#!/bin/bash
+
+VersionOfUbuntuContainer="22.04"
+
+function timestamp() {
+    echo "[+] $(date +'%F %T') [INFO] $*"
+    echo
+}
+
+function timestampEnding() {
+    echo "[+] $(date +'%F %T') [INFO] $*"
+}
+
+function err() {
+    echo "[-] $(date +'%F %T') [ERROR] $*" >&2
+    echo
+}
+
+function command_start() {
+    timestamp "Command $* has been started."
+    if ! "$@"; then
+        err "Command $* went wrong."
+        exit
+    fi
+    timestamp "Command $* has been ended."
+}
+
+function removeOldDockerContainers() {
+    docker stop "$(docker ps --all | grep -i Webi | awk '{print $1}')" >/dev/null
+    docker rm "$(docker ps --all | grep -i Webi | awk '{print $1}')" >/dev/null
+}
+
+function pullAndRunUbuntuContainer() {
+    docker pull jrei/systemd-ubuntu:$VersionOfUbuntuContainer
```

Comment:
Why should we trust jrei? Why not use the normal Ubuntu container?

Reply:
As far I remembered I tested packaged that required the systemd in the docker container. The default ubuntu container does not have that feature.